### DISCUSSION TITLE
Instances were predicted on 0 frames (20 frames with 0 instances found)

### DISCUSSION THREAD

**Discussion by CENICOL** [talmolab/sleap#2129] [Help!]
Hi! When I try to run training on a single animal I get the error "Instances were predicted on 0 frames (20 frames with no instances found)" and pages of error messages about not being able to find the videos. I am using 6 labels (nose, right forelimb, right backlimb, left forelimb, left backlimb, and tailbase), and I labeled around 10 frames per video, a total of 236 frames. My video files are .mp4 files.

I have attached the output during my training. Thank you for your assistance!

[Anaconda window from SLEAP training and run.docx](https://github.com/user-attachments/files/19034114/Anaconda.window.from.SLEAP.training.and.run.docx)

**Comment by gitttt-1234** [comment]
Hi @CENICOL ,

It seems like the model isn't learning (the loss doesn't seems to reduce through the epochs). This might be due to the size of receptive field, which is very crucial for single-instance models. The receptive field should almost be the size of the animal instance you want to detect. The model might find it hard to detect the animal, if receptive field (RF) is too small. 

Could you try increasing RF by increasing max stride to 32? Could you send a screenshot of training settings tab in the GUI (you could see the receptive field - blue box, highlighted in the left corner of the training settings tab)?

Thanks,

Divya

**Reply by CENICOL** [reply]
Hi Divya,

Thank you for your response. I modified the receptive field by increasing max stride to 32 but still got the same error at the end (output attached). I am including a screenshot of the training settings I am using (the max stride is set to 32, but the screenshot is from just before I did that). 

I also don't know if it would help the training if I cropped the frames I am using given the mirrors that are present in my apparatus?

[SLEAP output 2.docx](https://github.com/user-attachments/files/19114900/SLEAP.output.2.docx)
[Training settings.docx](https://github.com/user-attachments/files/19114909/Training.settings.docx)

Thank you for your help,

Nicole
